modules/apps/frontend-editor/frontend-editor-ckeditor-web/package.json

```diff
@@ -2,7 +2,7 @@
 	"dependencies": {
 		"ckeditor4-react": "1.0.1",
 		"frontend-js-web": "*",
-		"liferay-ckeditor": "4.14.1-liferay.2"
+		"liferay-ckeditor": "^4.14.1-liferay.3"
```

Comment:
Let's keep this exactly locked like we used to.

Reply:
Ups, I did `yarn add` and didn't notice it